doc/api/v8.md

```diff
@@ -1051,6 +1051,51 @@ added:
 
 Returns true if the Node.js instance is run to build a snapshot.
 
+### Class: `v8.MemoryProfileWatchdog`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+#### `new MemoryProfileWatchdog([options])`
+
+* `interval` {number} How often the watchdog polls.
+* `maxRss` {number} Node.js will take snapshot when rss is
+  greater than or equal to `maxRss`.
+* `maxUsedHeapSize` {number} Node.js will take snapshot when used heap size is
+  greater than or equal to `maxUsedHeapSize`.
+* `filename` {string} Node.js will write heap snapshot to `filename`.
+
+Creates a new `MemoryProfileWatchdog` object, `maxRss` or `maxUsedHeapSize` must
+be setted. `MemoryProfileWatchdog` will create a thread and poll the memory
+info periodically, when the rss or used heap size is greater than or equal to
+the value you set, Node.js will write a heap snapshot to `filename` then stop
+the watchdog.
+
+#### `memoryProfileWatchdog.stop()`
```

Comment:
The example isn't using `.stop()`

Reply:
Yes, the watchdog will be destroyed automatically when process/thread exits. I will add `stop` to example.

Reply:
Fixed.